src/shared/lib/geography/timezone.js

```diff
@@ -0,0 +1,27 @@
+import assert from 'assert';
+import iso2Codes from 'country-levels/iso2.json';
+import usStates from '../../vendor/usa-states.json';
+
+// eslint-disable-next-line import/prefer-default-export
+export const calculateScraperTz = async location => {
```

Comment:
Any reason this is async? Looks like sync operations to me, unless you're planning on async lookups for !USA?

Reply:
> unless you're planning on async lookups for !USA?

Yes, `src/shared/lib/geography/country-levels.js getLocationData` is async and will be used for !USA ones.